node/packages/aws-lambda-otel-extension/internal/otel-extension-internal-node/index.js

```diff
@@ -198,26 +198,23 @@ const responseHandler = async (span, { res, err }, isTimeout) => {
   if (isTimeout) {
     functionData.error = true;
     functionData.errorCulprit = 'timeout';
-    functionData.errorExceptionType = 'TimeoutError';
-    functionData.errorExceptionMessage = '';
-    functionData.errorExceptionStacktrace =
-      'Function execution exceeded the configured timeout limit.';
+    functionData.errorType = 'TimeoutError';
   } else if (err) {
     functionData.error = true;
     functionData.errorCulprit = err.message;
-    functionData.errorExceptionType = typeof err;
-    functionData.errorExceptionMessage = err.message;
-    functionData.errorExceptionStacktrace = err.stack;
+    functionData.errorType = 'handled';
```

Comment:
I think it'll be nice to have unified naming convention for _types_.

This PR proposes two types: `TimeoutError` and `handled` that doesn't look good.

What would be nice is `timeout` and `handled` but can we do that?

Reply:
I went ahead and updated the timeout `errorType` to `timeout` 😎